### PR TITLE
Gui 2411 grid view w/ separate graph page.

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2411

For info on the service introduced into landingpage.js, look at "lesson 2" here: http://onehungrymind.com/angularjs-sticky-notes-pt-1-architecture/

### CODE REVIEW

File: eucaconsole/static/js/pages/metrics.js [hunk 1]
```diff
@@ -156,11 +157,11 @@ angular.module('MetricsPage', ['LandingPage', 'CloudWatchCharts', 'EucaConsoleUt
             });
         };
         vm.sortGetters = {
-            resources: function(value) {
-                if (headResources === undefined) {
-                    headResources = $("tr>th:nth-of-type(2)");
+            resources: function(value, decending) {
+                if (decending === undefined) {
```
Comment (coyote240) [inline eucaconsole/static/js/pages/metrics.js]: descending

File: eucaconsole/views/metrics.py [hunk 2]
```diff
@@ -116,6 +116,12 @@
         resource=['AvailabilityZone', 'LoadBalancerName'],
     ),
```
Comment (coyote240) [inline eucaconsole/views/metrics.py]: I've been meaning to ask, why the tendency towards the dict() constructor and not simply {}?

File: eucaconsole/static/js/pages/metrics.js (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/static/js/pages/metrics.js]: I eyed that suspiciously as well.

File: eucaconsole/views/metrics.py (hunk 2 above)
Comment (dkavanagh) [inline eucaconsole/views/metrics.py]: no particular reason.

**Comment by dkavanagh** [comment]
hal, retest this please

**Pull request merged**